sec1_intro_and_setup/1.1_basics.rb

```diff
@@ -0,0 +1,17 @@
+#Section 1.1
+#Let's code the quiz!
+
+#Creating an array
+address = [1,2,3,4,5,6,7,8,9]
+
+#printing array
+p address
+
+#printing array with index 5
+p address[5]
+
+#assigning new_address to store the reverse of the array
+new_address = address.reverse!
```

Comment:
what does `!` does?

Reply:
@tlkiong The ! here is modifying the variable address in place, instead of returning the modified value as a new variable.

Reply:
@ziqiang2344 Ok. If thats the case, why equals it to `new_address`?

Reply:
In this case, the new_address is assigned to it to address. reverse! to experiment with the array variables assignment.

Reply:
1. `to experiment with the array variables assignment.` is shown in line 5 already
2. do we need to do this?

Reply:
Not necessarily. We can still print the reverse of the address without assigning it to the `new_address`.

Reply:
I have removed the `new_address` from the code.